src/Microsoft.AspNetCore.Mvc.Formatters.Json/MvcJsonOptionsExtensions.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.AspNetCore.Mvc
```

Comment:
This should be in the `Microsoft.Extensions.DependencyInjection` namespace along with `IMvcBuilder`.

Reply:
@AlexejTimonin would you be able to address these comments this week?

Reply:
@pranavkm Yep, I'll be able to so it.